t/page/needed.rb

```diff
@@ -9,7 +9,12 @@
   end
 
   it 'should know that Eritrea is missing' do
-    subject.to_find.map(&:title).include? 'Eritrea'
+    assert subject.to_find.map(&:title).include? 'Eritrea'
+  end
+
+  it 'should order the list of countries to find by name' do
+    order = subject.to_find.map(&:title).first <=> subject.to_find.map(&:title).last
+    assert order <= 0
```

Comment:
This is very complex logic to have in a test (and tests that effectively say "foo should be either X or Y" are a also a red flag — here you're saying it should be zero _or_ less than")

One way to express the intent more clearly might be something like:
```ruby
wanted = subject.to_find.map(&:title)
assert wanted == wanted.sort
```